### PR TITLE
test config readme

### PR DESCRIPTION
**Pull request by AnieeG**

### CODE REVIEW

**Review by kalverra — APPROVED** [review]

File: integration-tests/ccip-tests/testconfig/README.md [hunk 1]
```diff
@@ -0,0 +1,690 @@
...
+dashboard_url = "/d/6vjVx-1V8/ccip-long-running-tests"
+```
+
+## CCIP.Groups
+Specifies the test config specific to each test type. Available test types are:
+- **CCIP.Groups.load**
+- **CCIP.Groups.smoke**
+- **CCIP.Groups.chaos**
+
+### CCIP.Groups.<testtype>.KeepEnvAlive
```
Comment (b-gopalswami) [inline integration-tests/ccip-tests/testconfig/README.md]: Wherever we have this `<testtype>`, it is not rendering and it breaks the title reference in most of the places.

File: integration-tests/ccip-tests/testconfig/README.md [hunk 2]
```diff
@@ -0,0 +1,690 @@
...
+
+### CCIP.Groups.<testtype>.LocalCluster
+Specifies whether the test is to be run on a local docker. If set to true, the test environment will be set up on a local docker.
+
+### CCIP.Groups.<testtype>.ExistingDeployment
+Specifies whether the test is to be run on existing deployments. If set to true, the test will use the deployment data specified in [CCIP.Deployments](#ccipdeployments) for interacting with the ccip contracts.
+If the deployment data does not contain the required contract addresses, the test will fail.
+
+### CCIP.Groups.<testtype>.ReuseContracts
+Test loads contract/lane config from [contracts.json](/integration-tests/ccip-tests/contracts/laneconfig/contracts.json) if no lane config is specified in [CCIP.Deployments](#ccipdeployments)
```
Comment (b-gopalswami) [inline integration-tests/ccip-tests/testconfig/README.md]:
```suggestion
Test loads contract/lane config from [contracts.json](./../contracts/laneconfig/contracts.json) if no lane config is specified in [CCIP.Deployments](#ccipdeployments)
```

File: integration-tests/ccip-tests/testconfig/README.md [hunk 3]
```diff
@@ -0,0 +1,690 @@
...
+If a certain contract is present in the contracts.json, the test will use the contract address from the contracts.json.
+This field specifies whether to reuse the contracts from [contracts.json](/integration-tests/ccip-tests/contracts/laneconfig/contracts.json)
```
Comment (b-gopalswami) [inline integration-tests/ccip-tests/testconfig/README.md]:
```suggestion
This field specifies whether to reuse the contracts from [contracts.json](./../contracts/laneconfig/contracts.json)
```

**Review by b-gopalswami — COMMENTED** [review]
Thanks for adding this detailed document on test config which will be very useful for new comers.
Just couple of broken links and rendering issues are highlighted. Rest are all LGTM!

**Review by b-gopalswami — APPROVED** [review]

**Pull request merged**